### DISCUSSION TITLE
¿Free download of premium content?

### DISCUSSION THREAD

**Discussion by jjaruna** [streamlink/streamlink#5267] [Q&A]
### Description

hi everyone, do you know if there is a way to download the premium content of a video for free? From live.nicovideo.jp 

It won't let me buy the nicovideo membership and there is a video that disappears in a few days.

I hope no offense to anyone, but I do not know what to do lol

**Comment by bastimeyer** [comment]
- https://streamlink.github.io/plugins.html#nicolive
- https://streamlink.github.io/cli.html#file-output-options
